lib/kino_db/connection_cell.ex

```diff
@@ -383,7 +400,7 @@ defmodule KinoDB.ConnectionCell do
       Code.ensure_loaded?(Exqlite) -> "sqlite"
       Code.ensure_loaded?(ReqBigQuery) -> "bigquery"
       Code.ensure_loaded?(ReqAthena) -> "athena"
-      Code.ensure_loaded?(Adbc) -> "snowflake"
+      Code.ensure_loaded?(Adbc) -> "duckdb"
```

Comment:
This is tricky. Adbc supports duckdb now, but it also snowflake and soon bigquery. I wonder which one should really be the default?

Reply:
Three good arguments. :)